src/resources/material.js

```diff
@@ -178,8 +183,18 @@ class MaterialHandler {
         }
     }
 
+    _onTextureUnload(parameterName, materialAsset, textureAsset) {
+        const material = materialAsset.resource;
+        if (material) {
+            if (material[parameterName] === textureAsset.resource) {
+                this._assignTexture(parameterName, materialAsset, null);
```

Comment:
should this revert to placeholder texture or is null correct?

Reply:
I wasn't certain - but the test case didn't show any errors - so decided not to change too much  - there is a null check and assigning of placeholders in the bind function so I assumed placeholders get assigned in that code path.

Reply:
Found a case that would require placeholders - so now replace null with placeholders